### ISSUE TITLE
Downloads are setting a source value when sending to logger

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/biocache-service#429]
The logger stats are showing `0` counts for the ALA source (`sourceId: 0`):

https://logger.ala.org.au/service/sourceBreakdown?eventId=1002

![image](https://user-images.githubusercontent.com/532845/73796487-70a12e80-4801-11ea-985c-1a3fe281c303.png)

Investigate and fix this so ALA stats are being recorded.

Possible causes: `downloads-plugin` and `bioacache-service`.

**Comment by ansell** [comment]
Given that OZCAM and AVH are there, it is possibly an ala-hub configuration issue, which would flow through to downloads-plugin

**Comment by nickdos** [comment]
Yes, was thinking same thing - they are all using the same code so its likely config.
